### PR TITLE
Import action modules lazily

### PR DESCRIPTION
**Pull request by kouk**
This way we do not contribute to the import dependencies of any module
that will import `event.models`.

### CODE REVIEW

**Review by fathineos — APPROVED** [review]
Good one! 👍

File: events/event.py [hunk 1]
```diff
@@ -2,21 +2,35 @@
...
+                self._methods.append((action, method))
+        return self._methods
+
     def emit(self, *args, **kwargs):
         logger = logging.getLogger('events')
-        for l, action in self._listeners:
+        for name, method in self.methods:
             try:
-                l(action=action)(*args, **kwargs)
+                method(*args, **kwargs)
```
Comment (kouk) [inline events/event.py]: We should still instantiate the class every time the event is emitted.

Comment (fathineos) [reply events/event.py]: True, but this is sufficient to avoid circular dependencies when importing top level modules.

**Review by igavriil — APPROVED** [review]
Nice enchantment! Finally no circular dependency madness. :tada: :+1:

**Pull request merged**